### DISCUSSION TITLE
Marks Labels

### DISCUSSION THREAD

**Discussion by Bimbara28** [cepdnaclk/e19-co227-digital-reporting-of-technical-visits#78] [Q&A] [answered]
Chandula could you please tell me that, how to add marks labels to contributers of the repository?

**Comment by chandula00** [comment, marked as answer]
Ah, I introduced that feature as a way to allocate points to tasks depending on their significance or priority. It's a flexible system where you can simply select these labels and assign them to any task in our workflow. The idea behind this implementation is to enhance the enthusiasm and engagement of our team members when it comes to completing tasks. By attaching different weights to tasks, we not only recognize their varying importance but also create a gamified approach that encourages everyone to contribute their best efforts. 😊
